publication-commons/src/test/java/no/unit/nva/publication/permission/strategy/PublicationPermissionStrategyTest.java

```diff
@@ -648,6 +659,15 @@ protected List<AccessRight> getCuratorAccessRightsWithDegree() {
         return accessRights;
     }
 
+    protected List<AccessRight> getCuratorAccessRightsWithEmbargoDegree() {
```

Comment:
Kind of misleading name. Thought it meant returning all access rights with embargo degree, which would only be `MANAGE_DEGREE_EMBARGO`. 

Kind of risky to hard code these rights tho, as the single source of truth should be in identity service. But that might be a hassle.

```suggestion
    protected List<AccessRight> getAccessRightsForEmbargoThesisCurator() {
```

Reply:
yup, as long as identity-service owns the role definitions, doing something fancy is probably not worth it. We could maybe think about what should be included or not here